spec/shopify_transporter/pipeline/magento/product/variant_image_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'shopify_transporter/pipeline/magento/product/variant_image'
+
+module ShopifyTransporter::Pipeline::Magento::Product
+  RSpec.describe VariantImage, type: :helper do
```

Comment:
Add a test for when there are multiple images? I believe that the code path that concatenates images onto an existing image array is not being tested.

Reply:
![image](https://user-images.githubusercontent.com/22937651/46164831-05cf0300-c25d-11e8-881b-ab03edf7d5d9.png)

We need to make sure this file is 100% covered